backend/heartcode/src/main/java/com/itvitae/heartcode/evaluation/EvaluationController.java

```diff
@@ -0,0 +1,32 @@
+package com.itvitae.heartcode.evaluation;
+
+import com.itvitae.heartcode.match.MatchService;
+import com.itvitae.heartcode.user.User;
+import com.itvitae.heartcode.user.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/v1/evaluations")
+@CrossOrigin("http://localhost:5173")
```

Comment:
het cross-origin-allowed-origins address staat in application.properties, dat is ook nodig om de api op je telefoon te bereiken